bcl2fastq/index_summary.py

```diff
@@ -152,12 +191,23 @@ def update_known_index_bar(run_json):
     [dep.Input('pruned_unknown_hidden', 'children')]
 )
 def update_unknown_index_pie(run_json):
+    """ When input (sample_run_hidden) is changed, function is rerun to get update bar graph unknown indices
```

Comment:
This docstring says it causes a bar graph to change, but to me it looks like the name of the method implies that a pie chart changes instead. Can these be harmonized?

Reply:
I'm not sure why the name has the word "pie", the data actually does return the bar graph at the bottom of the report